src/hooks/bookmarks/use-all-bookmarks.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery, UseQueryResult } from '@tanstack/react-query'
+import { ApiBookmark } from 'lib/learn-client/api/api-types'
+import { getAllBookmarks } from 'lib/learn-client/api/bookmark'
+import useAuthentication from 'hooks/use-authentication'
+
+interface UseAllBookmarksResult extends Omit<UseQueryResult, 'data'> {
```

Comment:
This is mainly just renaming `data` to `bookmarks` so the property has a clearer name that can make the hook consumption a little clearer. I can update it to reference `data` though, rather than remove + set our own.